sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpProperty.cs

```diff
@@ -37,12 +37,6 @@ internal static class AmqpProperty
         ///
         public static AmqpSymbol Timeout { get; } = AmqpConstants.Vendor + ":timeout";
 
-        /// <summary>
-        ///   The identifier of the consumer to associate with a link.
-        /// </summary>
-        ///
-        public static AmqpSymbol ConsumerIdentifier { get; } = AmqpConstants.Vendor + ":receiver-name";
```

Comment:
In JS, we have the concept of a id/name that is assigned to the link, but we dont let the user configure it.
It gets used in the logs mainly and we have found it useful to determine what happened to which link when.
Am not sure how logging is done in .Net and this might not be of any concern for you, but just putting it out here

Reply:
I don't think it's needed, as it would only impact the service message coming back, and the value would be meaningless to users.

Our link is strongly associated with an identifier for the transport client that it uses, which holds responsibility for logging.   This impacts the message from the service, which is wrapped in our own logging metadata.   The end result is that our logs will report the "we made it up" identifier, Event Hub name, and other stuff wrapped around all of our messages.

Users looking over the .NET logs will see the sequence for "Client XXX started,  Client XXX had an error,  Client XXX completed"  for our operations.